Podfile.lock

```diff
@@ -69,6 +70,7 @@ DEPENDENCIES:
   - SocketRocket (from `https://github.com/square/SocketRocket.git`, commit `954750c018`)
   - SSKeychain
   - TwistedOakCollapsingFutures (~> 1.0)
+  - UICKeyChainStore (from `Podspecs/UICKeyChainStore.podspec`)
```

Comment:
I had to add this back in to do the migration as this is where data is stored for Signal on the app store